README.md

```diff
@@ -1,113 +1,167 @@
-# Impress
-
-Impress is a web application for real-time collaborative text editing with user and role based access rights.
-Features include : 
-- User authentication through OIDC
-- BlocNote.js text editing experience (markdown support, dynamic conversion, block structure, slash commands for block creation) 
-- Document export to pdf and docx from predefined templates
-- Granular document permissions
-- Public link sharing
-- Offline mode
-
-Impress is built on top of [Django Rest Framework](https://www.django-rest-framework.org/), [Next.js](https://nextjs.org/) and [BlocNote.js](https://www.blocknotejs.org/)
-
-## Getting started
-
-### Prerequisite
-
-Make sure you have a recent version of Docker and [Docker
-Compose](https://docs.docker.com/compose/install) installed on your laptop:
-
-```bash
+<p align="center">
+  <a href="https://github.com/suitenumerique/docs">
+    <img alt="Docs" src="/docs/assets/logo-docs.png" width="300" />
+  </a>
+</p>
+
+<p align="center">
+Welcome to Docs! The open source document editor where your notes can become knowledge through live collaboration
+</p>
+
+<p align="center">
+  <a href="https://matrix.to/#/#docs-official:matrix.org">
+    Chat on Matrix
+  </a> - <a href="/docs/">
+    Documentation
+  </a> - <a href="#getting-started">
+    Getting started
+  </a>
+</p>
+
+<img src="https://github.com/suitenumerique/docs/blob/main/docs/assets/docs_live_collaboration_light.gif?raw=true" width="600" align="center"/>
+
+## Why use Docs ❓
+Docs is a collaborative text editor designed to address common challenges in knowledge building and sharing.
+
+### Write
+*   😌 Simple collaborative editing without the formatting complexity of markdown
+*   🔌 Offline? No problem, keep writing, your edits will get synced when back online
+*   💅 Clean formatting options to focus on content creation and create beautiful documents
```

Comment:
and creating?

Reply:
How about?
* 💅 Create clean documents with limited but beautiful formatting options and focus on content